### PR TITLE
Update catering.md

### PR DESCRIPTION
**Pull request by pcai**
integrating some of my recent experience as a first time catering coordinator.

### CODE REVIEW

File: catering.md [hunk 1]
```diff
@@ -1,30 +1,55 @@
 # Catering
 
 Catering is the biggest budget item for RailsBridge and a major part of the
-event. We typically do the following, based on 80-90 people at the workshop:
+event. We typically do the following, based on 50-60 people at the workshop:
```
Comment (edwardloveall) [inline catering.md]: I first read this and though "No way do we have that many people show up" but then I remembered TAs and Organizers exist. Maybe it makes sense to mention all of the groups of people so future caterers keep that in mind?

Comment (edwardloveall) [reply catering.md]: I see you have some implicit mention of that below, but I still think it might read well if mentioned here in the total number.

**Comment by mxie** [comment]
@pcai Ping! Any updates here? Just want to get this merged in before I reorganize this file. :)

File: catering.md (hunk 1 above)
Comment (pcai) [reply catering.md]: updated to clarify, how's that look?

Comment (edwardloveall) [reply catering.md]: Pefect!

**Pull request merged**